css/design_system/typography.scss

```diff
@@ -0,0 +1,108 @@
+/**
+ * @title Heading 1
+ * @category typography
+ * @value .s-h1
+ * @order 1
```

Comment:
We shouldn't have to manually define the order here. They should be ordered by how they are written within the scss file